server/handler/status.go

```diff
@@ -120,8 +120,8 @@ func (h *Status) processOthers(ctx context.Context, event github.StatusEvent) er
 		ownerName,
 		repoName,
 		commitSHA,
-		&github.PullRequestListOptions{
-			ListOptions: github.ListOptions{PerPage: 100},
+		&github.ListOptions{
```

Comment:
This was a legitimate compile break introduced by an upstream change https://github.com/google/go-github/pull/2822